src/cbmc/cbmc_parse_options.cpp

```diff
@@ -804,6 +812,21 @@ int cbmc_parse_optionst::get_goto_program(
 
   goto_model = initialize_goto_model(cmdline.args, ui_message_handler, options);
 
+  if(cmdline.isset("use-rra"))
+  {
+    std::string abst_file = options.get_option("use-rra");
+
+    abstraction_spect abst_info(abst_file, ui_message_handler);
+
+    std::vector<std::string> abstfiles =
+      abst_info.get_abstraction_function_files();
+
+    goto_modelt goto_model_for_abst_fns =
+      initialize_goto_model(abstfiles, ui_message_handler, options);
```

Comment:
`use-rra` needs a re-initialize and re-link?

Reply:
This code is removed now.